examples/tools/generate.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Download this notebook :download:`here <generate.ipynb>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Generate data samples"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "There are two different types of data samples: (1) :ref:`phase space samples\n",
+    "<usage/tools/generate:Phase space samples>` and (2) :ref:`intensity-based\n",
+    "samples <usage/tools/generate:Intensity-based samples>` (which mimick the\n",
+    "intensity profile of the decay topology). In addition, you may want to\n",
```

Comment:
intensity profile of the decay topology -> intensity distribution of the reaction

Reply:
https://github.com/ComPWA/pycompwa/pull/79/commits/bd3e48cd35c29baaceb365f7afc72fa9e4139abb